joblib/memory.py

```diff
@@ -10,33 +10,29 @@
 
 
 from __future__ import with_statement
+
+import functools
+import inspect
+import io
+import json
```

Comment:
@lesteve I just checked again and `memory.py` uses pickle so that's why it's being modified.